templates/etc/etcd/etcd.conf.erb

```diff
@@ -62,6 +62,12 @@ ETCD_CERT_FILE="<%= scope['etcd::cert_file'] %>"
 <% unless [nil, :undefined, :undef, ''].include?(scope['etcd::key_file']) -%>
 ETCD_KEY_FILE="<%= scope['etcd::key_file'] %>"
 <% end -%>
+<% unless [nil, :undefined, :undef, ''].include?(scope['etcd::client_cert_file']) -%>
```

Comment:
in the ruby above the field defaults to false, which is not checked against here, allthough I assume it'll be fine since none of the other fields do